src/snowflake/snowpark/functions.py

```diff
@@ -567,7 +592,13 @@ def bround(col: ColumnOrName, scale: Union[Column, int]) -> Column:
     """
     col = _to_col_if_str(col, "bround")
     scale = _to_col_if_lit(scale, "bround")
-    return call_builtin("ROUND", col, scale, lit("HALF_TO_EVEN"))
+
+    ast = proto.Expr()
+    build_fn_apply(ast, "bround", col, scale)
+
+    col = call_builtin("ROUND", col, scale, lit("HALF_TO_EVEN"))
```

Comment:
Nit: Capitalization.

I assume identifier canonicalization happens at the server, so there's not much point in being strict about this at the client. Is that reasonable?

Reply:
left it as in the original snowpark python code. Could be lowercased. For now, added a comment.